docs/cloud/1-services/2-lightdb/3-read-write-data.md

```diff
@@ -168,3 +183,45 @@ The final state of the device in this scenario would be like this:
   }
 }
 ```
+
+### Listening to any real-time device state change via WebSocket
+
+Differently from GET method at which you receive one data at a time, with WebSocket it's possible to keep a persistent flow of data coming from the LightDB at the moment the state is changed. In other words, with WebSocket you are able to listen to changes in real-time.
+
+<!-- To open the WebSocket connection you will make a HTTP Request for an websocket endpoint passing the following parameteres:
+
+|Parameter | Optional | Description |
+|--------- | -------- | ----------- |
+|`ProjectID` | **true** | ID of the project to be listened |
+|`DeviceID` | **true** | ID of the device to be listened |
+|`Path` to listened to | false | _empty path_ if desired to listen to the root path <br /> _some path_ to listen to specific path |
+|`API_KEY` | **true** | Api Key to authenticate to the Golioth  | -->
+
+For example, for listening to any changes in the `/env` path, connect a WebSocket client to the following endpoint:
+```
+wss://api.golioth.net/v1/ws/projects/local-test/devices/6173155307bbb1c7c9bb158f/data/env?x-api-key=DDb2UQDZXmbFOp3duDm0CThtmQ9RJuPP
```

Comment:
be careful putting real/valid API Keys. Also let's change the URL to `api.golioth.io` which is the production URL.